### PR TITLE
remove unnecessary tests

### PR DESCRIPTION
**Pull request by ngriffiths13**
Ignoring a set of slow tests by default and some other small cleanup

### CODE REVIEW

File: tests/test_preprocessing.py [hunk 1]
```diff
@@ -340,16 +340,16 @@ def test_fractional_diff(pd_X):
 
 
 ### Temporarily commented out. Uncomment when benchmarking is ready. ###
```
Comment (baggiponte) [inline tests/test_preprocessing.py:342]: I guess you can remove this comment too

**Comment by baggiponte** [comment]
Hey there, thanks for the PR. The proposed solution is to test less regressors for the moment? It can work right now to speed up the feedback loop.

I am not super expert on our internals but I think in the long run we should add to our roadmap to refactor the tests again to run forecasters on smaller datasets (eg the commodities data? or a fake one with 10 series 100 observations) to make sure they actually fit. What do you think?

@topher-lo too I'd love to hear your opinion on this. As we mentioned in the discord too, tests run too slow and fail (after 3/4 hours). We could run the benchmarks before every release or in a separate repo like polars does.

**Comment by topher-lo** [comment]
We can disable most of the expensive datasets

**Comment by topher-lo** [comment]
Also would recommend splitting up feature extraction and forecasting tests into two separate workflows.

**Comment by baggiponte** [comment]
Do you think we should do this now? Can we change the test dataset to commodities? How do we know what set of tests we can run on each PR?

**Pull request merged**